userguide/storage.rst

```diff
@@ -1789,22 +1790,30 @@ Replacing an Encrypted Disk
 If the ZFS pool is encrypted, additional steps are needed when
 replacing a failed drive.
 
-First, make sure that a passphrase has been set using the instructions
-in :ref:`Managing Encrypted Pools` **before** attempting to replace
-the failed drive. Then, follow steps 1 and 2 as described above.
-During step 3, there will be a prompt to enter and confirm the
-passphrase for the pool. Enter this information, then click
-:guilabel:`REPLACE DISK`.
-
-Wait until resilvering is complete before
-:ref:`restoring the encryption keys to the pool <Managing Encrypted Pools>`.
-**Restore the encryption keys before the next reboot or access to
-the pool will be permanently lost**.
-
-#.  Highlight the pool that contains the recently replaced disk
-    and click :guilabel:`Add Recovery Key` to save the new
-    recovery key. The old recovery key will no longer function, so it
-    can be safely discarded.
+Ensure a :ref:`passphrase has been set <Encryption Options>`
+**before** attempting to replace the failed drive. To replace a drive
+in an encrypted pool
+
+#. Go to 
+   :menuselection:`Storage --> Pools -->` |ui-settings| :menuselection:`--> Status`.
```

Comment:
Would it be useful to insert an extra instruction after `Pools`?

``--> Pools. Locate the encrypted pool entry and click |ui-settings| :menuselection:`--> Status`.``